README.md

```diff
@@ -58,6 +58,13 @@ This application can be either run as a web application or a standalone desktop
 
 To run, `npm start` or `yarn start`.
 
+If preferred, a Docker alternative is available:
+```
+docker-compose build
+docker-compose up
+```
+Then, open a browser to `http://localhost:3000`.
```

Comment:
Probably worth adding that users should run `docker-compose down` after exiting docker to shut down running containers.

Reply:
Done.